src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -66,7 +66,7 @@ const propTypes = {
         avatar: PropTypes.string,
     }),
 
-    /** Currently viewed reportID */
+    /** Currently viewed reportID. This value is parsed from the react navigation state object */
     currentlyViewedReportID: PropTypes.string,
```

Comment:
NAB: It might actually be better to either:

1. Pass down the entire route object, just like this had access to `routes` in the first place
2. Rename this to something a little better like `reportIDFromURL`

As you pointed out in one of your comments, the report isn't always "visible".

Reply:
Oh I see what you mean since "currently viewed" implies that someone is viewing it but the report itself could be "out of view" though it would still be accurate to say this is a "report ID from the URL" without any possibility to interpret the "visibility" part.

Reply:
Went with option 2 but changed it to `reportIDFromRoute` since feels more accurate on native